### DISCUSSION TITLE
How to inspect(or list) stream consumer info offline?

### DISCUSSION THREAD

**Discussion by emacsist** [rabbitmq/rabbitmq-server#6444] [Questions] [answered]
`rabbitmq-streams list_stream_consumers` , but it need consumer online. 

how to inspect stream consumer info (or list all consumer info) even though consume is offline ?

**Comment by kjnilsson** [comment, marked as answer]
Try something like this:

rabbitmq-streams stream_status stream-name --tracking

On Tue, 22 Nov 2022 at 12:36, emacsist ***@***.***> wrote:

> rabbitmq-streams list_stream_consumers , but it need consumer online.
>
> how to inspect stream consumer info (or list all consumer info) even
> though consume is offline ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rabbitmq/rabbitmq-server/discussions/6444>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJAHFD3OARJKUEGO3WORDDWJS44ZANCNFSM6AAAAAASHYVUOE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Karl Nilsson*

**Reply by emacsist** [reply]
oh, thanks, that's what i need.
